Patch to make the make_UNIX.inc file EasyBuild compatible
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/SLICOT-Reference.orig/make_Unix.inc b/SLICOT-Reference/make_Unix.inc
index 0139fc1..6aa4dc3 100644
--- a/SLICOT-Reference.orig/make_Unix.inc
+++ b/SLICOT-Reference/make_Unix.inc
@@ -13,13 +13,14 @@ SHELL = /bin/sh
 #  and desired compiler options for your machine.  NOOPT refers to
 #  the compiler options desired when NO OPTIMIZATION is selected.
 #
-FORTRAN = gfortran
-OPTS = -O2 -fPIC -fdefault-integer-8
+FORTRAN  = ${FC}
+OPTS     = ${FCFLAGS} -fdefault-integer-8
+NOOPT    =
 OPTS_NOOPT = -O0
 
 #  Define LDFLAGS to the desired linker options for your machine.
 #
-LOADER   = $(FORTRAN) 
+LOADER   = ${FC}
 LDFLAGS =
 
 #  The archiver and the flag(s) to use when building an archive
@@ -36,7 +37,7 @@ LOADOPTS = $(SLICOTLIB) $(LPKAUXLIB) $(LAPACKLIB) $(BLASLIB)
 #  machine-specific, optimized BLAS library should be used whenever
 #  possible.)
 #
-BLASLIB      = ../libs/blas.a
-LAPACKLIB    = ../libs/lapack.a
+BLASLIB      = -L${EBROOTOPENBLAS}/lib -lflexiblas
+LAPACKLIB    = -L${EBROOTOPENBLAS}/lib -lflexiblas
 LPKAUXLIB    = ../lpkaux.a
 SLICOTLIB    = ../slicot.a
